scalardb-test/src/main/java/kelpie/scalardb/transfer/graphql/HttpTransferProcessor.java

```diff
@@ -75,156 +89,99 @@ public void executeEach() throws Exception {
     int toId = ThreadLocalRandom.current().nextInt(numAccounts);
     int amount = ThreadLocalRandom.current().nextInt(1000) + 1;
 
-    logStart(fromId, toId, amount);
-
-    String txId;
-    try {
-      txId = transfer(fromId, toId, amount);
-    } catch (Exception e) {
-      logFailure(fromId, toId, amount, e);
-      throw e;
-    }
-    logSuccess(txId, fromId, toId, amount);
+    transfer(fromId, toId, amount);
   }
 
   @Override
   public void close() {}
 
-  private String transfer(int fromId, int toId, int amount) throws Exception {
-    int fromType = 0;
-    int toType = 0;
-    if (fromId == toId) {
-      toType = 1; // transfer between the same account
-    }
-
-    JsonObject variables1 =
-        Json.createObjectBuilder()
-            .add("from_id", fromId)
-            .add("from_type", fromType)
-            .add("to_id", toId)
-            .add("to_type", toType)
-            .build();
-    JsonObject response1 = sendRequest(GET_BALANCES, variables1);
-
-    JsonArray errors = response1.getJsonArray("errors");
-    if (errors != null) {
-      throw new GraphQlFailureException("", errors);
-    }
+  private void transfer(int fromId, int toId, int amount) throws Exception {
+    try {
+      int fromType = 0;
+      int toType = 0;
+      if (fromId == toId) {
+        toType = 1; // transfer between the same account
+      }
 
-    String txId =
-        response1.getJsonObject("extensions").getJsonObject("transaction").getString("id");
-    JsonObject data = response1.getJsonObject("data");
-    int fromBalance =
-        data.getJsonObject("from_account")
-            .getJsonObject(TransferCommon.TABLE)
-            .getInt(TransferCommon.BALANCE);
-    int toBalance =
-        data.getJsonObject("to_account")
-            .getJsonObject(TransferCommon.TABLE)
-            .getInt(TransferCommon.BALANCE);
-
-    JsonObject variables2 =
-        Json.createObjectBuilder()
-            .add("from_id", fromId)
-            .add("from_type", fromType)
-            .add("to_id", toId)
-            .add("to_type", toType)
-            .add("tx_id", txId)
-            .add("from_new_balance", fromBalance - amount)
-            .add("to_new_balance", toBalance - amount)
-            .build();
-    JsonObject response2 = sendRequest(PUT_BALANCES, variables2);
+      JsonObject variables1 =
```

Comment:
It's not for this PR, but I feel creating several JsonObject instances for each request might be too heavy (since JsonObject is pretty slow compared with the other Json libraries), causing unnecessarily slow performance for GraphQL.
For comparison purposes, it shouldn't be a big problem, though.

Reply:
Thank you for the feedback. Then we should change it to use a different Json library in the future. I will add it to my to-do list. Thanks.